app/controllers/licensees_controller.rb

```diff
@@ -1,6 +1,67 @@
 # frozen_string_literal: true
 
 class LicenseesController < ApplicationController
+
+    before_action :authenticate_user!
+    
+    def new
+        @licensee = Licensee.new
+    end
+
+    def create
+        Licensee.create(
+            facility_name: params[:licensee][:facility_name],
+            contact_name: params[:licensee][:contact_name],
+            address: params[:licensee][:address],
+            city: params[:licensee][:city],
+            state: params[:licensee][:state],
+            zip: params[:licensee][:zip],
+            phone: params[:licensee][:phone],
+            fax: params[:licensee][:fax],
+        )
+        redirect_to licensees_path
+    end
+
+    def index
+        @licensees = Licensee.all
+        # @q = Licensee.ransack(params[:q])
+        # @licensees = @q.result(distinct: true)
+    end
+    
+    def show
+        @licensee = Licensee.find(params[:id])
+    end
+    
+    def edit
+        @licensee = Licensee.find(params[:id]) 
+    end
+
+    def update
+        @licensee = Licensee.find(params[:id])
+        @licensee.facility_name = params[:licensee][:facility_name]
+        @licensee.contact_name = params[:licensee][:contact_name]
+        @licensee.address = params[:licensee][:address]
+        @licensee.city = params[:licensee][:city]
+        @licensee.state = params[:licensee][:state]
+        @licensee.zip = params[:licensee][:zip]
+        @licensee.phone = params[:licensee][:phone]
+        @licensee.fax = params[:licensee][:fax]
+        @licensee.save
+        redirect_to licensee_path(params[:id])
+    end
+    
+    def destroy
+         @licensee = Licensee.find(params[:id])
+         @licensee.destroy
+         redirect_to licensees_path
+    end
+
+#   def index
+#       @q = Licensee.ransack(params[:q])
+#       @licensees = @q.result(distinct: true)
+#   end
+    
+
   before_action :authenticate_user!
   before_action :authorize_admin, only: %i[new edit]
 
```

Comment:
It looks like this file contains a duplicate of itself. Please correct this so that the file only contains one version of each method.

This could be something that happened as a result of keeping both versions of a merge conflict. If so, you'll just need to select which version should be kept and delete the half of the file that should be removed.